### PR TITLE
Correct usage of this.$$

### PR DESCRIPTION
**Pull request by arthurevans**
Fixes Polymer/polymer#2693.

### CODE REVIEW

File: 1.0/docs/devguide/local-dom.md [hunk 1]
```diff
@@ -226,9 +226,9 @@ You can also locate nodes in the local DOM using `querySelector`, `querySelector
 
     var cancelButton = Polymer.dom(this.root).querySelector('#cancelButton');
 
-    // Shorthand for finding local DOM children by ID
+    // Shorthand for finding local DOM children by selector
     // (equivalent to the above):
-    this.$$('cancelButton');
+    this.$$('#cancelButton');
```
Comment (ebidel) [inline 1.0/docs/devguide/local-dom.md]: i wonder if a class would be a better example. It's usually assumed only a single node on the page has an ID.

File: 1.0/docs/devguide/local-dom.md [hunk 2]
```diff
@@ -226,9 +226,9 @@ You can also locate nodes in the local DOM using `querySelector`, `querySelector
 
     var cancelButton = Polymer.dom(this.root).querySelector('#cancelButton');
```
Comment (ebidel) [inline 1.0/docs/devguide/local-dom.md]: this should be querySelectorAll right? $$ returns more than one node.

Comment (arthurevans) [inline 1.0/docs/devguide/local-dom.md]: I believe it only returns one:

https://github.com/Polymer/polymer/blob/master/src/standard/utils.html#L28

File: 1.0/docs/devguide/local-dom.md [hunk 3]
```diff
@@ -226,9 +226,9 @@ You can also locate nodes in the local DOM using `querySelector`, `querySelector
...
 
-    // Shorthand for finding local DOM children by ID
+    // Shorthand for finding local DOM children by selector
```
Comment (arthurevans) [inline 1.0/docs/devguide/local-dom.md]: Maybe I should change this to 'a local DOM child'?

File: 1.0/docs/devguide/local-dom.md (hunk 2 above)
Comment (ebidel) [inline 1.0/docs/devguide/local-dom.md]: Ah you're. Man, that's pretty confusing. Literally everyone uses `$$` for qSA

**Comment by ebidel** [comment]
LGTM then

**Comment by arthurevans** [comment]
Corrected comment, merging.

**Pull request merged**